### ISSUE TITLE
[gatsbyjs.org] Website does not display in Firefox on Freebsd XFCE.

### ISSUE THREAD

**Issue by dwagnkorea** [gatsbyjs/gatsby#4830]
## Description

The gatsbyjs.org website does not display in firefox on freebsd xfce.

### Steps to reproduce

1) Install FreeBSD with XFCE as the display manager
2) Open Firefox
3) Browse to gatsby.org

### Expected result

The website should display something

### Actual result

The website displays for a second, and then dissappears...

### Environment

* Gatsby version (`npm list gatsby`): None
* gatsby-cli version (`gatsby --version`): None
* Node.js version: None
* Operating System: FreeBSD 10.3-RELEASE-p28 XFCE

### File contents (if changed):

None

**Comment by dwagnkorea** [comment]
I will note that this is preventing me from even getting started using Gatsby.  This seems like a serious problem for a tool that claims to be able to generate web pages....

**Comment by m-allanson** [comment]
What version of Firefox are you using?

**Comment by janjarfalk** [comment]
The issue is re-creatable in Firefox 39 (Windows/OSX) and lower due to usage of `Array.prototype.includes`.

```TypeError: n.includes is not a function```

**Comment by sebastienfi** [comment]
Release is from 2015. Do we really want to include a polyfill for such an old chap ?

**Comment by KyleAMathews** [comment]
We could perhaps polyfill it 🤷‍♂️ it also doesn't work on IE. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

I'm going to move the website over to Gatsby v2 pretty soon though which will automatically polyfill things like this based on the browsers we support so thanks for raising this issue @dwagnkorea! I'll close this now as it'll be fixed in a few days.

**Comment by sebastienfi** [comment]
Gatsby V2 releases in a few days ? That's a big news @KyleAMathews ahahahah

**Comment by KyleAMathews** [comment]
Moving the website over :-) v2 release still a bit out. The more immediate milestone is moving the website over, releasing a beta, and making v2 the default branch.

**Comment by dwagnkorea** [comment]
I determined that my the version where I saw this issue is Firefox Version 35.0.1.

I would propose that Browser and Browser Version should be questions in a bug report, since that appears to be important in cases such as this one.

Well, perhaps you wonder why I am using version 35?  The reason is that 35 is the highest version supported by FreeBSD Version 10.3.  I made many attempts to upgrade it and read through a ton of documentation before I finally determined that Firefox cannot be upgraded beyond this in my OS.

Then perhaps you wonder why I am using FreeBSD 10.3, instead of 11.1, or even 10.4?

Well, version 10.3 is still supported, although it will soon reach it's end of support.  I don't necessarily want to reinstall my OS and all my Applications every time there is a new version released.  I generally don't want to do that as long as my current OS is still supported.  This is not Ubuntu or Windows, where I can just push a button to upgrade to the latest OS and everything will be fine.

  Also, Version 10.3 was released on April 4 2016, so it is barely 2 years old.  That's even newer than Windows 10.  I use FreeBSD so that I can mostly forget about the OS.  I can leave it running for days or even months and I don't need to reinstall anything, or even reboot my computer.  It doesn't seem unreasonable to expect your website to work on a 2 year old OS which is still supported.  

  So should  I upgrade my OS and reinstall all my Applications, just so that I can view the Gatsby.org website?  Well, given that FreeBSD 10.3 will soon reach end of support, I might be willing to do that.  However, given that Gatsby is a tool designed to build websites, this means I have to expect the same of anyone visiting any website I build with Gatsby.

  So, anyone with a one or two year old browser won't even be able view the website I make with Gatsby.  Really?  That includes some of my friends who still use Windows XP.  That includes my friends who still use Android 3.0 and possibly 4.0.  That includes everyone using Firefox OS itself.  Should I complain to them?  So should I invest the time to even try out Gatsby?  Really?

  Well, it looks like the website renders for a second and then disappears.  So it seems like it should not be hard to fix.  Just stop it from disappearing.  But I guess that's up to the Gatsby devs.
